composer.json

```
@@ -125,7 +125,7 @@
     },
     "scripts": {
         "check-cs": "@fix-cs --dry-run",
-        "fix-cs": "php-cs-fixer fix -v --show-progress=dots",
+        "fix-cs": "php-cs-fixer fix --config=.php-cs-fixer.php --show-progress=dots",
```

`-v` is needed by `--show-progress=dots`
```suggestion
        "fix-cs": "php-cs-fixer fix -v --config=.php-cs-fixer.php --show-progress=dots",
```